assets/javascripts/discourse/services/chat.js

```diff
@@ -545,6 +548,27 @@ export default Service.extend({
     });
   },
 
+  _subscribeToChannelStatusChange() {
+    this.messageBus.subscribe("/chat/channel-status", (busData) => {
```

Comment:
I'm not sure if you can do it (I swear I thought I saw you do it before) but is there any way we can test this?

One day if we remove that `window.location.reload`, the person who can make the change can hitch on the test.

Reply:
Actually yeah I think I can test it, however we usually wrap `window.location.reload` with `if (!isTesting())`, so I will fix that, but there would be nothing really to test then